dockerfile

```diff
@@ -0,0 +1,22 @@
+#Deriving the latest base image
+FROM node:latest
+
+WORKDIR /usr/app/babysleepcoach
+
+RUN mkdir -p video
+
+#Copy all files in the container
+COPY . .
+
+# Install required packages
+RUN apt-get update
+RUN apt-get update && apt-get install python3-pip libgl1  -y
+RUN pip3 install --upgrade pip
+RUN pip3 install -r requirements.txt
+
+RUN cd webapp && yarn install
+
+ENV SLEEP_DATA_PATH=/usr/app/babysleepcoach
+ENV VIDEO_PATH=/usr/app/babysleepcoach/video
+
+CMD ["./start_docker.sh"]
```

Comment:
When I try to run a container from built image, it errors trying to find this file. This works for you on your end?

Reply:
Sorry, I broke something before the pull request. I have added a necessary library. Now it should start.